### ISSUE TITLE
@WilliamJamieson needs admin access to asdf-astropy

### ISSUE THREAD

**Issue by pllim** [astropy/astropy-project#246]
### Description

@WilliamJamieson is now the lead maintainer of asdf-astropy. His work is greatly impacted by not having admin access to that repo. Thanks.

cc @nden

**Comment by adrn** [comment]
We were asked previously to give him maintainer privileges in #239 -- as far as I can tell, he does have maintainer access. Is this a request to update that to "admin" access?

**Comment by pllim** [comment]
He is unable to set up RTD without admin access. Is there a reason why he should not be given access as admin to the package that he leads?

**Comment by adrn** [comment]
Not at all! I just wanted to make sure I understood correctly -- updating privileges now!

**Comment by adrn** [comment]
Done

**Comment by adrn** [comment]
Actually: I previously just added him individually to the repo with admin access, but I think it makes sense to give anyone in @astropy/asdf-astropy-maintainers admin access, so I will do that now.

**Comment by pllim** [comment]
@adrn , can you please replace `eslavich` with @WilliamJamieson on the `asdf-astropy-maintainers` team? Thanks!

**Comment by hamogu** [comment]
@WilliamJamieson is on the asdf-astropy-maintainers team, https://github.com/orgs/astropy/teams/asdf-astropy-maintainers/members, as is eslavich. All team members of that team have (as team members) admin access to repro asdf-astropy (in addition, Plim and William are listed separately with access permissions "triage" and "admin" respectively, but in practice that should mot matter, because, as asdf-astropy-maintainers  team members, they have admin access anyway).

Thus, I'm not quite sure what the request is. Should eslavich  be removed? Also note that, @pllim (and all the others on that team) have team role "maintainer" in the asdf-astropy-maintainers team, so you should be able to manage team membership yourself.

At the very least, it might make sense, to remove individual repro permissions and only manage them through the team, but I'm not touching that right now until I have confirmation that I'm interpreting the request correctly.

**Comment by hamogu** [comment]
OK, so I've followed what @adrn and I suggested above and removed individual maintainers from that repro. Instead, all members of the team asdf-astropy-maintainers has admin access through team permissions and current team members can manage team membership themselves.

**Comment by WilliamJamieson** [comment]
Alright, I managed to get zenodo hooked up to `asdf-astropy`. Thanks
